Gulpfile.js

```diff
@@ -265,6 +278,124 @@ gulp.task('test-client-travis', ['build'], function () {
+    function lintFiles (files) {
+        return new Promise(function (resolve, reject) {
+            var config = {
+                'files':  files,
+                'config': require('./.markdownlint.json')
+            };
+
+            markdownlint(config, function (err, result) {
+                var lintErr = err || result && result.toString();
+
+                if (lintErr)
+                    reject(lintErr);
+                else
+                    resolve();
+            });
+        });
+    }
+
+    return globby('./docs/articles/**/*.md').then(lintFiles);
+});
+
+gulp.task('clear-site', ['lint-docs'], function (cb) {
+    del('./site/**', function (e) {
+        cb(e);
+    });
+});
+
+gulp.task('fetch-assets-repo', ['clear-site'], function (cb) {
+    git.clone('https://github.com/VasilyStrelyaev/testcafe-gh-page-assets.git', { args: './site' }, function (e) {
+        if (e) return cb(e);
+        cb();
+    });
+});
+
+gulp.task('put-in-articles', ['fetch-assets-repo'], function () {
+    return gulp.src('./docs/articles/**/*')
+		.pipe(gulp.dest('./site/src'));
+});
+
+gulp.task('put-in-navigation', ['put-in-articles'], function () {
+    return gulp.src('./docs/_data/**/*')
+		.pipe(gulp.dest('./site/src/_data'));
+});
+
+gulp.task('put-in-templates', ['put-in-navigation'], function () {
+    return gulp.src('./docs/_includes/**/*')
+		.pipe(gulp.dest('./site/src/_includes'));
+});
+
+gulp.task('put-in-docs', ['put-in-articles', 'put-in-navigation', 'put-in-templates']);
+
+gulp.task('prepare-website', ['fetch-assets-repo', 'put-in-docs']);
+
+gulp.task('build-website', ['prepare-website'], function (cb) {
+    var cp = spawn('jekyll.bat', ['build', '--source', './site/src/', '--destination', './site/deploy']);
+
+    cp.on('exit', function (code) {
+        cb(code === 0 ? null : 'ERROR: Jekyll process exited with code: ' + code);
+    });
+});
+
+gulp.task('serve-website', ['build-website'], function () {
+    server = connect().use('/testcafe', serveStatic('./site/deploy')).listen(8080);
+    server.on('connection', function (socket) {
+        sockets.push(socket);
+        socket.on('close', function () {
+            sockets.splice(sockets.indexOf(socket), 1);
+        });
+    });
+});
+
+gulp.task('open-website', ['serve-website'], function () {
+    opn('http://localhost:8080/testcafe');
+});
+
+gulp.task('prompt-web-server-close', ['open-website'], function (cb) {
+    var rl = readline.createInterface({
+        input:  process.stdin,
+        output: process.stdout
+    });
+
+    rl.question('Press Enter to terminate the web server...', function () {
+        server.close(function (e) {
+            if (e) return cb(e);
+            cb();
+        });
+        rl.write('Server stopped.\r\n');
+        sockets.forEach(function (socket) {
+            socket.destroy();
+        });
+        rl.close();
+    });
+});
+
+gulp.task('preview-website', ['prompt-web-server-close']);
```

Comment:
Why we need `prompt-web-server-close` task since it's all the same as `preview-website`?